src/sonic-frr/patch/0033-bgpd-backpressure-cleanup-bgp_zebra_XX-func-args.patch

```diff
@@ -171,21 +171,21 @@ index 69240a3b83..920df835a4 100644
  {
  	struct zapi_route api;
  	struct peer *peer;
+-
```

Comment:
why the whitespace change again?

Reply:
The patch is auto generated from the commit by "git format-patch -1" command.

Reply:
Here, the whitespace is actually removed.